ios/Handlers/LoginHandler.swift

```diff
@@ -29,6 +29,10 @@ class LoginHandler: NSObject {
     }
     
     func handleDidReceiveChallenge(_ challenge: ONGPinChallenge) {
+        // We need to cancel the active pinChallenge if we somehow received a new one from a different api.
+        if let pinChallenge = self.pinChallenge {
+            pinChallenge.sender.cancel(pinChallenge)
```

Comment:
No return after this? Should in this case the method proceed?

Reply:
This is actually doesn't seem to be required as the sdk does not send us a pinChallenge when we initiate a pin authentication while there is still another ONGPinChallenge active. 
There does however seem to be some weird behaviour when we initate an authenticateUser during authenticateUser/changepin flows which we will pick up in another story.